app/javascript/app/pages/mitigation/mitigation-actions/mitigation-actions-selectors.js

```diff
@@ -2,12 +2,11 @@ import { createSelector, createStructuredSelector } from 'reselect';
 import isEmpty from 'lodash/isEmpty';
 import { deburrUpper } from 'app/utils';
 
-const getInventoryData = ({ GHGInventory = {} }) =>
-  isEmpty(GHGInventory.data) ? null : GHGInventory.data;
+const getMitigationData = ({ mitigationActions = {} }) =>
+  isEmpty(mitigationActions.data['mitigation::MitigationActions'])
```

Comment:
for me `mitigationActions` already implies that the data in there is actually -> `data['mitigation::MitigationActions']`. So I would move that part to the provider itself or some kind of response serializer? what are your thoughts about this?

Reply:
This will be changed after the API changes are merged it will be mitigationActions.data